bundler/lib/bundler/cli/outdated.rb

```diff
@@ -166,46 +201,18 @@ def by_group(gems, filter: nil)
       end.compact
     end
 
-    def print_gems(gems_list)
+    def parseably_print_gems(gems_list)
```

Comment:
I can't quite get used to parsing this word "parseably". Does `parseable_print_gems` seem acceptable? Or if you have another idea.

Reply:
My fingers had trouble typing it correctly too >.<

`parseable_print_gems` doesn't have the right .. conjugation? I may be using that word wrong. Naming is hard though - `print_gems` is called for its side-effects, so it probably should be named with a verb-phrase, but I'm not too picky which one - `print_gems_carefully`, `print_gems_in_legacy_style`, `print_gems_manually`, `print_gems_locally`, `print_gems_for_parsing`? I'll use `print_gems_as_porcelain` for now, and update to whatever you prefer when you respond :-)

(My actual instinct was to extract that behavior into an inner class called `LegacyOutputGenerator` which would suffice to contextualize the printing instead, but it felt like too large a change for a PR that isn't purely refactoring. And I'm not sure how the maintainers feel about that sort of change yet)